flow-base-api/src/main/java/com/aevi/sdk/flow/stage/ServiceComponentDelegate.java

```diff
@@ -122,6 +128,20 @@ private void completeMessageStream() {
         return helper;
     }
 
+    @Nullable
+    public static String getActivityRequestJson(Activity activity) {
```

Comment:
Not a major thing - but not sure these really belong as static methods in this class?